Jenkinsfile.sonatype.bump-up-api

```
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+ * Includes the third-party code listed at http://links.sonatype.com/products/nexus/attributions.
+ * "Sonatype" is a trademark of Sonatype, Inc.
+ */
+@Library(['private-pipeline-library', 'jenkins-shared', 'int-jenkins-shared']) _
+
+node('ubuntu-zion') {
+  def platformApiVersion, branch
+
+  stage('Preparation') {
+    deleteDir()
+
+    // Checkout and config
+    checkout scm
+    runSafely 'git config user.email sonatype-ci@sonatype.com'
+    runSafely 'git config user.name Sonatype CI'
+
+    // Setup common variables
+    platformApiVersion = getVersionFromBuildName(env.platformApiBuild_NAME)
+    branch = 'main'
+  }
+  stage('Bump Nexus Platform API Version') {
+    mvn jenkinsMavenConfig(), 'org.codehaus.mojo:versions-maven-plugin:2.8.1:set-property -DgenerateBackupPoms=false ' +
+      "-DnewVersion=${platformApiVersion} -Dproperty=nexus-platform-api.version"
+  }
+  stage('Build & Test') {
+    mvn jenkinsMavenConfig(), 'clean verify'
+  }
+  stage('Archive Results') {
+    junit '**/target/surefire-reports/TEST-*.xml'
+    archive 'target/*.hpi'
+  }
+  stage('Push code and tags') {
```

```suggestion
  stage('Push code') {
```
As far as I can tell there is no tagging done here.